src/main/java/ch/wisv/areafiftylan/controller/TeamRestController.java

```diff
@@ -142,6 +144,41 @@ public Team getTeamById(@PathVariable Long teamId) {
     }
 
     /**
+     * Add members to the Team with the given Id. Expects only a username as Requestbody. Can only be done by the
+     * Captain or an Admin
+     *
+     * @param teamId   Id of the Team to which the User should be added.
+     * @param username Username of the User to be added to the Team
+     *
+     * @return Result message of the request
+     */
+    @PreAuthorize("@currentUserServiceImpl.canEditTeam(principal, #teamId) " +
+            "and @currentUserServiceImpl.hasTicket(#username)")
+    @RequestMapping(method = RequestMethod.POST, value = "/{teamId}/invites")
+    public ResponseEntity<?> inviteTeamMember(@PathVariable Long teamId, @RequestBody String username) {
+        teamService.inviteMember(teamId, username);
+        return createResponseEntity(HttpStatus.OK, "User " + username + " successfully invited to Team " + teamId);
+    }
+
+    @PreAuthorize("@currentUserServiceImpl.canEditTeam(principal, #teamId)")
+    @RequestMapping(method = RequestMethod.GET, value = "/{teamId}/invites")
+    public List<TeamInviteResponse> getTeamInvitesByTeam(@PathVariable Long teamId) {
+        return teamService.findTeamInvitesByTeamId(teamId);
+    }
+
+    @RequestMapping(method = RequestMethod.POST, value = "/invites")
```

Comment:
Team Captain can accept the invites he sends since he can retrieve the token. I would add Authentication.